.github/workflows/release.yml

```diff
@@ -0,0 +1,72 @@
+name: Release
+
+on:
+  push:
+    branches: [main]
+
+concurrency: ${{ github.workflow }}-${{ github.ref }}
+
+permissions: {}
+
+jobs:
+  release:
+    permissions:
+      contents: write # to create release
+      issues: write # to post issue comments
+      pull-requests: write # to create pull request
+
+    if: github.repository == 'trpc/trpc'
+
+    timeout-minutes: 20
+
+    name: Release
+
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      matrix:
+        node: ['18.x']
+        os: [ubuntu-latest]
+
+    env:
+      TURBO_TOKEN: ${{ secrets.TURBO_TOKEN }}
+      TURBO_TEAM: ${{ secrets.TURBO_TEAM }}
+
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v3
+
+      - uses: pnpm/action-setup@v2.2.4
```

Comment:
I'm not super familiar with pnpm - I just copy-pasted this stuff from the `main.yml`

Reply:
I think we need a version script that will sync the lockfile:

See https://github.com/changesets/changesets/issues/421